msprime/demography.py

```diff
@@ -705,6 +705,42 @@ def add_census(self, time: float) -> CensusEvent:
         """
         return self.add_event(CensusEvent(time))
 
+    def activate_population(self, time: float, population: Union[int, str]):
```

Comment:
We're not going to be adding this event to the public API, so I don't think we need the method here. We can just use the classes manually  within the code (or, make this a private method)